samples/typescript/KafkaTriggerRetryExponential/function.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is the file name correct?

Reply:
captured in -- https://github.com/Azure/azure-functions-kafka-extension/pull/376